iOS/JSQMessagesViewController/build.cake

```diff
@@ -3,6 +3,14 @@
 
 var TARGET = Argument ("t", Argument ("target", "Default"));
 
+var IOS_PODS = new List<string> {
```

Comment:
Do not fetch/build the native library this way.  Take a look at the `externals/Makefile` . It fetches the code, makes some changes and builds the library locally.